src/main/java/com/google/api/codegen/transformer/java/JavaGapicSurfaceTransformer.java

```diff
@@ -68,7 +72,7 @@
   private ApiMethodTransformer apiMethodTransformer;
   private PageStreamingTransformer pageStreamingTransformer;
   private BundlingTransformer bundlingTransformer;
-  private ImportTypeTransformer importTypeTransformer;
+  private FileHeaderTransformer fileHeaderTransformer;
   private RetryDefinitionsTransformer retryDefinitionsTransformer;
```

Comment:
A general question: Should these be turned into static finals to help signify that they don't hold state?